content/en/security/sensitive_data_scanner/guide/redact_uuids_in_logs.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Redact Universal Unique IDs (UUIDs) in Logs
+disable_toc: false
+further_reading:
+- link: "/security/sensitive_data_scanner/scanning_rules/custom_rules"
+  tag: "Documentation"
+  text: "Regex for custom scanning rules"
+---
+
+## Overview
+
+This guide goes over how to create a custom scanning rule using a regular expression (regex) pattern that matches and redacts a Universally Unique Identified (UUID). For example, your organization has a UUID that is for internal identification only, and the UUID has additional information appended to it, such as the user's:
+- User ID
+- Department code
+- Status code
+
+If you want internal users to have access to these logs but don't want to expose the UUID and user ID, you can create a custom scanning rule to redact the information.
+
+## Set up a custom rule to match a UUID
+
+For this guide, `01e2402104ca99-8641-43ba-b499-642610-0012` is the example internal identifier being used, where:
+- `01e2402104ca99-8641-43ba-b499` is the UUID.
+- `6462610` is a 6-digit value that represents the ID in byte format.
+- `0012` is A 2-digit department code and 2-digit status code of a user:
+    - `00` is used for an active user.
+    - `12` for the department code.
+
+In this example, you want to match the format of the example identifier (`01e2402104ca99-8641-43ba-b499-642610-0012`) and:
+- Redact the UUID, user ID, and the ID in byte format.
+- But not redact the department and status code of the user.
+
+You can use the following basic regex to match the UUID and user ID that you want to redact:
+
+```
+[a-z0-9]{14}-\d{4}-[a-z0-9]{4}-[a-z0-9]{4}-\d{6}
+```
```

Comment:
````suggestion
```

This regex pattern captures:
- The 14-character alphanumeric start of the UUID
- A sequence of structured segments (`-####-####-####`)
- The 6-digit byte-format user ID

It excludes the final 4-digit department and status code, allowing that information to remain visible.

````